server/health_status.go

```diff
@@ -0,0 +1,9 @@
+package server
+
+type HealthStatus struct {
+	HealthStatus string `json:"healthStatus" binding:"required"`
+}
+
+const OK = "OK"
```

Comment:
You don't need to use variables here, `string` is ok.
or a new Enum would be better if you insist.